python_modules/libraries/dagster-dbt/dagster_dbt/asset_decorator.py

```diff
@@ -34,16 +38,21 @@ def dbt_assets(
     Examples:
         .. code-block:: python
 
-            from dagster import OpExecutionContext
-            from dagster_dbt import DbtCli, DbtManifest, dbt_assets
-
-            manifest = DbtManifest.read(path="target/manifest.json")
+            from pathlib import Path
 
+            from dagster import OpExecutionContext
+            from dagster_dbt import DbtCli, dbt_assets
 
-            @dbt_assets(manifest=manifest)
+            @dbt_assets(manifest=Path("target", "manifest.json"))
```

Comment:
Might have to check the other docstrings in `resources_v2` to make sure its now following this new convention